example/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,19 @@
+<resources>
+    <!-- Main Strings -->
+    <string name="app_name">Ejemplos de AppIntro</string>
+    <string name="drawer_header">Cabecera del Drawer</string>
+    <string name="launch_demo">Iniciar Demo</string>
+
+    <!-- 1st Tab Group -->
+    <string name="default_intro">Este es el ejemplo más sencillo posible de la librería AppIntro. Puede llevarse a cabo con muy pocas líneas de código, y está disponible con dos layouts distintos. Pulsa en el botón de abajo para iniciar la demo.</string>
+    <string name="default_intro2">Este es el segundo ejemplo de la librería AppIntro. Puede llevarse a cabo con muy pocas líneas de código, y este es el segundo tipo de layout que ofrece la librería. Pulsa en el botón de abajo para iniciar la demo.</string>
+    <string name="custom_layout_intro">Aquí se demuestra como se puede usar un layout personalizado en la librería para tener un mayor nivel de personalización. Pulsa en el botón de abajo para iniciar la demo.</string>
+    <string name="custom_background_intro">This demonstrates the ability to set custom backgrounds for different slides. Useful if you want images vs. colors in your backgrounds. Tap the button below to launch the demo.</string>
+    <string name="custom_background_intro">Aquí se demuestra como se puede poner un fondo personalizado para cada slide. Esto es util si quieres usar colores contrastando con imagenes para tus fondos. Pulsa en el botón de abajo para iniciar la demo.</string>
```

Comment:
this is duplicated, I think it can bring a compilation error, please leave the translated one.